services/osparc-gateway-server/tests/integration/test_dask_sidecar.py

```diff
@@ -101,8 +101,8 @@ async def _wait_for_service_to_be_ready(
     "image_name",
     [
         "itisfoundation/dask-sidecar:master-github-latest",
-        "itisfoundation/dask-sidecar:staging-github-latest",
-        "itisfoundation/dask-sidecar:release-github-latest",
+        # "itisfoundation/dask-sidecar:staging-github-latest",
```

Comment:
comments?? I expected more from you sylvain ;-))))

Reply:
right... let me delete them